lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -537,21 +554,25 @@ void OpenCLFunction::execute() {
     if (I.isDataParallel() && !isa<CopyInst>(I)) {
       // Figure out how many element-wise elements are there to process:
       size_t global;
-      if (I.isDataParallel()) {
-        global = I.getOperand(0).first->getType()->size();
-        if (global % 16 == 0) {
-          // Start less kernels and let each kernel do more work using vector
-          // instructions.
-          global /= 16;
-          kernelName += "16";
-        } else if (global % 8 == 0) {
-          // Start less kernels and let each kernel do more work using vector
-          // instructions.
-          global /= 8;
-          kernelName += "8";
+      if (!isQuantized) {
```

Comment:
May be move this check below `global = I.getOperand(0).first->getType()->size();` to make things a bit simpler.

Reply:
cool